tasks/upload_processor.py

```diff
@@ -256,7 +240,7 @@ def process_impl_within_lock(
                         repoid=repoid,
                         commit=commitid,
                         arguments=arguments,
-                        commit_yaml=commit_yaml.to_dict(),
+                        commit_yaml=commit_yaml,
```

Comment:
So the commit_yaml we're logging here is the one we pass in the function definition, but before it was the one you got rid off in here, https://github.com/codecov/worker/pull/592/files#diff-4d7b2da9b8b9cf3e98040654f5510214c375f699ccafe41558b2f37519174110L212. Do you know if it's safe to delete the `commit_yaml = UserYaml(commit_yaml)` line? Could we mistakenly be logging/using the incorrect value?

Reply:
As far as I traced the code, there was only one user of the `UserYaml` form, and I inlined that.
Though this change assumes that `dict_form == UserYaml(dict_form).to_dict()` holds, which I’m not 100% it does?
In either case, the dict form seems to be only used for logging, which I guess is not as critical.